scripts/install/deploy/install_redis.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# @todo #240:30min Implement redis availability check and install flows. In case
+# Redis is not available lets install it and launch.
```

Comment:
@ekondrashev The puzzle is not well formatted: you should add a leading space for all lines AFTER the @todo

Reply:
@emilianodellacasa fixed, thx